build/LLVM.lua

```diff
@@ -74,6 +74,9 @@ function SetupLLVMLibs()
 
   configuration "macosx"
     links { "c++", "curses", "pthread", "z" }
+    
+  configuration "vs2015"
+    links { "version" }
```

Comment:
This is required by Clang, not by VS, so I think it's not just VS 2015 that needs it. @tritao what do you think?